sdk/python/feast/infra/offline_stores/offline_store.py

```diff
@@ -78,6 +78,8 @@ def to_df(
         if self.on_demand_feature_views:
             # TODO(adchia): Fix requirement to specify dependent feature views in feature_refs
             for odfv in self.on_demand_feature_views:
+                if odfv.mode != 'pandas':
```

Comment:
@maksstach 
We will want to fix this later on. Main implication is that `get_historical_features` won't work (I think) with Python transformations.